test/bump.test.js

```diff
@@ -190,7 +231,7 @@ tap.test('should work with a custom version-prefix', async () => {
   sinon.assert.calledWithExactly(stubs.execWithOutputStub, 'git', [
     'commit',
     '-m',
-    `Release v${TEST_VERSION}`,
+    `Release ${TEST_VERSION}`,
```

Comment:
why would this assertion be different from what it was before?

Reply:
I think this test was passing as a false positive previously - on the `main` branch in the nearform-actions repo if you run it as the only (or first) test in the file it fails and, following the code through, it shouldn't have the `v` if you pass `inputs['version-prefix']` as `''`.

Having a look into why this is happening now, feels like maybe the mocks aren't getting restored/reset properly after each tests

Reply:
yes please let's spend some time trying to figure out what's going on here to avoid surprises